unity/CAVE/Assets/W3D_Translator/CLI.cs

```diff
@@ -200,11 +214,9 @@ private static void BuildWalls(Story xml, Transform storyT)
     }
 
     // Convert Story.ObjectRoot to a dictionary of {name: GameObject} pairs
-    private static Dictionary<string, GameObject> TranslateGameObjects(
-        List<W3D.Object> objectList, GameObject story
-    )
+    private static ObjDictionary TranslateGameObjects(List<W3D.Object> objectList, GameObject story)
     {
-        Dictionary<string, GameObject> gameObjects = new();
+        ObjDictionary gameObjects = new();
```

Comment:
Why do you need this special class if there's no special methods for the new class?

Reply:
Looks like I can use a `using` directive? https://stackoverflow.com/questions/12099271/simplify-building-dictionaries

Reply:
That's why I like type inference so much! 

```
var gameObjects = new Dictionary<string, GameObject>();
```

You've got the ugly definition once, but the compiler tracks it for the rest of the life of that variable

Reply:
> Looks like I can use a `using` directive? https://stackoverflow.com/questions/12099271/simplify-building-dictionaries

Interesting! I've never seen `using` used like that!